morpheus/loaders/file_to_df_loader.py

```diff
@@ -88,7 +88,8 @@ def file_to_df_loader(control_message: ControlMessage, task: dict):
     parser_kwargs = config.get("parser_kwargs", None)
     cache_dir = config.get("cache_dir", None)
 
-    downloader = Downloader()
+    if not hasattr(file_to_df_loader, "downloader"):
+        file_to_df_loader.downloader = Downloader()
```

Comment:
I think it would be better to make the dask cluster static on the `Downloader` class which will avoid a possible race condition here if this is run from multiple threads